polymer/foam-select.html

```diff
@@ -0,0 +1,138 @@
+<link rel="import" href="./bower_components/polymer/polymer.html">
+
+<dom-module id="foam-select">
+  <script>
+  Polymer({
+    is: 'foam-select',
+
+    properties: {
+      dao: Object,
+      data: {
+        type: Array,
+        notify: true
+      }
+    },
+
+    observers: [
+      '_onDataUpdate(data.*)'
+    ],
+
+    ready: function() {
+      this._ids = {};
+
+      this._sink = {
+        put: this._put.bind(this),
+        remove: this._remove.bind(this),
+        reset: this._reset.bind(this),
+        eof: function() {}
+      };
+
+      this.feedback_ = true;
+      this.data = [];
+      this.feedback_ = false;
+
+      this.dao = eval(this.dao);
```

Comment:
It looks like you could change the type of the dao property to string and do this.get(this.dao, window);

https://www.polymer-project.org/1.0/api/#Polymer.Base:method-get